frontend/src/app/modules/dashboard/views/chat/components/chat-contacts/components/chat-contact-item/chat-contact-item.component.scss

```diff
@@ -1,10 +1,12 @@
+@import 'colors';
+
 :host {
   &.selected {
     .chat-contact-item {
-      background-color: #eaf3ff;
+      background-color: $chat-bg-color-select;
```

Comment:
Daca sa zicem ca e o exceptie de culoare, gen ai pus-o specific $chat-bg-color atunci pe nimeni nu mai intereseaza si ar trebui sa fie sus aici sub importuri variabila declarata , nu in _colors.scss

Reply:
fixed